### ISSUE TITLE
vulp tails and syndie hardsuits

### ISSUE THREAD

**Issue by rb303** [ParadiseSS13/Paradise#10305]
**Problem Description**:
the syndicate blood-red hardsuit makes tail stick out for vulps

**What did you expect to happen**: 
for my tail to fit inside the little bag thingy

**What happened instead**:
my tail sticks out and is exposed to the vacuum of space

**Why is this bad/What are the consequences:**
muh poor tail gonna smell like space

**Steps to reproduce the problem**:
put on syndie blood-red hardsuit (this was without the helmet, it probably does the same thing with the helmet on too but still)
**When did the problem start happening**:
only noticed it yesterday, i thought this bug was fixed a while ago

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was anything else out of the ordinary happening?)**:
it was the hardsuit found at one of the space ruins (the one that doesn't come with a helmet because skrek)
![image](https://user-images.githubusercontent.com/25644617/48857478-e235be00-edb0-11e8-97ed-2473c8188489.png)
![image](https://user-images.githubusercontent.com/25644617/48857499-eeba1680-edb0-11e8-8284-887a5fd484f6.png)

**Comment by KasparoVy** [comment]
Huh. That suit should havr the hidetail flag on it which would sort this issue. If it does have the flag and this issue persists, icon generation has the problem

**Comment by AzuleUtama** [comment]
Alright, so I went and did some testing myself and did not experience the same graphical issues you seem to be getting.

Unless I'm mistaken, there's only one version of the blood red hardsuit (minus the shield version, though I did also test this). I tried equipping this multiple times on a test server and didn't get any of the issues above. This was done both with and without a helmet.

There's seemingly nothing wrong in the code either. As KasparoVy mentioned the hidetail flag is inherited correctly along with all other hardsuits.

Not entirely sure where to go from here I'm afraid.

**Comment by rb303** [comment]
Hm, might've been a one time thing, I'll see if I can replicate it myself later.

**Comment by Kyep** [comment]
In my PR https://github.com/ParadiseSS13/Paradise/pull/10253/files I modified update_icons.dm.
It used to have a hardcoded check so that all items under /obj/item/clothing/suit/space would be treated as if they had HIDETAIL even when they did not.
I removed that check, allowing tails to show when wearing hardsuits, UNLESS that flag was added to the particular suit.
So, if this bug is happening, that's probably the cause, and the fix might be adding HIDETAIL to the suit.
